api/src/queries/project/project-delete-queries.ts

```diff
@@ -38,3 +38,75 @@ export const deleteIUCNSQL = (projectId: number): SQLStatement | null => {
 
   return sqlStatement;
 };
+
+/**
+ * SQL query to delete project focal species rows.
+ *
+ * @param {projectId} projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const deleteFocalSpeciesSQL = (projectId: number): SQLStatement | null => {
+  defaultLog.debug({
+    label: 'deleteFocalSpeciesSQL',
+    message: 'params',
+    projectId
+  });
+
+  if (!projectId) {
+    return null;
+  }
+
+  const sqlStatement: SQLStatement = SQL`
+    DELETE
+      from focal_species
+    WHERE
+      p_id = ${projectId}
+    RETURNING
+      *;
```

Comment:
Is this needed? What does it even return?

Reply:
Will find out shortly as Im just adding the delete bit for project activities/climate

Reply:
yeah that can be removed now - I was trying some stuff but yea * returns all the rows deleted which doesn't really have to be returned rn